ApiDoctor.Console/Program.cs

```diff
@@ -1854,7 +1854,7 @@ private static async Task<bool> CheckServiceMetadataAsync(CheckMetadataOptions o
                         {
                             RelaxedStringValidation = true,
                             IgnorablePropertyTypes = metadataValidationConfigs?.IgnorableModels,
-                            AllowTruncatedResponses = modelConfigs?.SkipProprtiesValidation ?? false
+                            AllowTruncatedResponses = modelConfigs?.TruncatedProprtiesValidation ?? false
```

Comment:
NIT: Missing letter `TruncatedPropertiesValidation`

Reply:
Thanks for noticing this. As this is a sync pr, I'm not going to make any changes.
We could address that in a separate pr, or pr the upstream. Do you want to create an issue here so we tackle that before merging back to upstream?

Reply:
https://github.com/OneDrive/apidoctor/issues/101